Content.Server/GameTicking/GameTickerCommands.cs

```diff
@@ -244,6 +244,19 @@ public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
         }
     }
 
+    class ToggleDisallowLateJoinCommand: IClientCommand
+    {
+        public string Command => "toggledisallowlatejoin";
+        public string Description => "";
+        public string Help => "";
+
+        public void Execute(IConsoleShell shell, IPlayerSession player, string[] args)
+        {
+            var ticker = IoCManager.Resolve<IGameTicker>();
+            ticker.ToggleDisallowLateJoin(bool.Parse(args[0]));
```

Comment:
You should only execute the command if you actually have the arguments (check the length of the args array and return if it's not enough)